applications/asset_tracker/src/main.c

```diff
@@ -528,6 +537,99 @@ void sensors_start(void)
 	}
 }
 
+/**@brief nRF Cloud specific callback for cloud association event. */
+static void on_user_pairing_req(const struct cloud_event *evt)
+{
+	if (evt->data.pair_info.buf!= NULL) {
```

Comment:
You could check the `type` here in order to resolve whether to do the pattern matching or pin pairing. IMO that would be easier to understand.

Reply:
Yeah, forgot to update that when the new struct was added. Will fix, thanks